join.html

```diff
@@ -394,12 +394,9 @@ <h2 class="progress">Here is your personalized link</h2>
           <ol>
             <li>Copy the above link</li>
             <li>
-              Follow Ankush Dharkar on
-              <a href="https://twitter.com/ankushdharkar">Twitter</a>
-            </li>
-            <li>
-              Message this link to him with a text of Real Dev Squad
-              Verification Link.
+              Message this link to Ankush on
```

Comment:
can we change it to ->`Message this link to Ankush over direct message over Twitter`

Reply:
i don't think it's needed.

Reply:
nit: it is, try to ask other people on what could be the text here

Reply:
text is good Ankush agrees